buildSrc/src/main/groovy/io.deephaven.bin-conventions.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java'
+}
+
+// our "bin" projects are a little special; the resources that go into the jar
```

Comment:
Should we rename this "configs"? "resources"? Should we try to conform to IntelliJ's expectations?

Reply:
I'm doing one better. I'm restructuring the "bin" projects, and deleting the custom logic that was previously used for them.